### PR TITLE
feat(logging): pretty print received and sent bidi server messages

### PR DESCRIPTION
**Pull request by thiagowfx**
Example: Screenshot 2023-10-27 at 11 05 56

Instead of having the output message string in a single line.

### CODE REVIEW

File: src/bidiServer/WebSocketServer.ts [hunk 1]
```diff
@@ -157,7 +156,16 @@ export class WebSocketServer {
         }
 
         const plainCommandData = message.utf8Data;
-        debugRecv(plainCommandData);
+        try {
+          const plainCommandDataPrettyPrint = JSON.stringify(
```
Comment (Lightning00Blade) [inline src/bidiServer/WebSocketServer.ts:160]: No need to `strigify` it back I think, let the debug handle it.

**Review by Lightning00Blade — APPROVED** [review]
LGTM, note that this will have have some marginal impact on performance as you always parse the string, I am not sure where this is used, but I think we only use it for test, Puppeteer and ChromeDriver don't need it, right? If we use it in ChromeDriver we should think of mitigating it.

**Comment by thiagowfx** [comment]
> performance

Hmmm, maybe we could do something like "if (logging_on) {..." to skip the parsing when there is no logging.

File: src/bidiServer/WebSocketServer.ts (hunk 1 above)
Comment (thiagowfx) [reply src/bidiServer/WebSocketServer.ts]: Very good, it even adds colors to the JSON output!

**Comment by thiagowfx** [comment]
https://www.npmjs.com/package/debug#checking-whether-a-debug-target-is-enabled

**Comment by thiagowfx** [comment]
re: performance: updated, PTAL

**Pull request merged**